ta/os_test/sub.mk

```diff
@@ -20,3 +20,11 @@ cxxflags-remove-cxx_tests.cpp-y += -pg
 srcs-y += cxx_tests_c.c
 cflags-remove-cxx_tests_c.c-y += -pg
 endif
+
+ifeq ($(sm),ta_arm64)
```

Comment:
Depending on `CFG_TA_PAUTH` being `y` should be enough.

Reply:
Ok.

How's the sm set to ta_arm64 ? I tried using CFG_USER_TA_TARGETS=ta_arm32, but that doesn't seem to set this. i remember this worked when i had tested the same with BTI.

Reply:
In `export-ta_arm64/mk/conf.mk`. You select the devkit correspondingly.

Reply:
Only CFG_TA_PAUTH is not enough.  PAC would work only with arm64 TA's. If we compile with COMPILE_S_USER=32, issues will be observed in compilation.

Reply:
It seems a bit strange to have a feature enabled which isn't supported even at build time.
But there isn't much we can do about that here. Let's keep it as it is for now.

Reply:
Yes, we did discuss this when BTI was being added. This seemed the most feasible solution.